src/utils/clm_time_manager.F90

```diff
@@ -496,11 +502,12 @@ end subroutine timemgr_restart_io
 
   !=========================================================================================
 
-  subroutine timemgr_restart( )
+  subroutine timemgr_restart()
 
     !---------------------------------------------------------------------------------
     ! Restart the ESMF time manager using the synclock for ending date.
     !
+
```

Comment:
The purpose of this subroutine is now, just to do some checking, set a couple variables, and to do the advance.

Reply:
Moving this as a thing to do in #2912